pkgs/games/path-of-building/default.nix

```
@@ -0,0 +1,67 @@
+{ stdenv, lib, fetchFromGitHub, runCommand, unzip, meson, ninja, pkg-config, qtbase, qttools, wrapQtAppsHook, luajit }:
+let
+  dataVersion = "2.27.0";
+  frontendVersion = "unstable-2023-04-09";
+in
+stdenv.mkDerivation {
+  pname = "path-of-building";
+  version = "${dataVersion}-${frontendVersion}";
```

```suggestion
  version = dataVersion;
```
that seems to be the relevant version number here

---

I mostly want frontend updates to be visible in the version number, because they affect things more than they possibly should.